testsuite/MDAnalysisTests/analysis/test_waterdynamics.py

```diff
@@ -91,13 +98,26 @@ def test_MeanSquareDisplacement_zeroMolecules(universe):
     assert_almost_equal(msd_zero.timeseries[1], 0.0)
 
 
-def test_SurvivalProbability(universe):
-    sp = MDAnalysis.analysis.waterdynamics.SurvivalProbability(universe,
-                                                               SELECTION1,
-                                                               0, 6, 3)
+def test_SurvivalProbability(universe_protein):
+    # The first timepoint has to be '1' since it is an autocorrelation function
+    sp = MDAnalysis.analysis.waterdynamics.SurvivalProbability(
+        universe_protein, SELECTION3, 0, 10, 4)
     sp.run()
-    assert_almost_equal(sp.timeseries[1], 1.0,
-                        decimal=5)
+    assert_equal(sp.timeseries[0], 1.0)
+
+
+def test_SurvivalProbability_t0Ignored(universe_protein):
```

Comment:
this test isn't very good, all it checks is that t0 makes a difference.  I'd replace it with a regression test, ie run with t0=3 manually, note down the result(s) and then hardcode these results into the test

will also help with performance as there's only one analysis done

Reply:
That's a good point. I did not want to use explicit numbers because I thought I would imply that they are correct per se (there was no verification against other established packages). 

I'll update it and resubmit it this weekend.